tox.ini

```diff
@@ -27,3 +27,10 @@ whitelist_externals=echo
 commands=
         python setup.py check -r -s
         echo setup.py/long description is syntaxly correct
+
+[testenv:bandit]
+basepython=python3.7
```

Comment:
Perhaps remove this so that it also runs on machines w/o Python 3.7.

Reply:
So it would pick whatever python version available?

Reply:
That was my thinking indeed...